h11/_readers.py

```diff
@@ -236,7 +236,7 @@ def __call__(self, buf):
                 return None
             matches = validate(chunk_header_re, chunk_header)
             # XX FIXME: we discard chunk extensions. Does anyone care?
-            self._bytes_in_chunk = int(matches["chunk_size"], base=16)
+            self._bytes_in_chunk = int(bytes(matches["chunk_size"]), base=16)
```

Comment:
I had to stare at this for a bit (I think it must be because our ReceiveBuffer actually returns bytearrays rather than bytes, and regex matching on a bytearray actually returns bytearray groups, which is surprising but whatever?). So, a comment would be good :-)